mmv1/products/workflows/terraform.yaml

```diff
@@ -23,8 +23,9 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       update_minutes: 6
       delete_minutes: 6
     autogen_async: true
-    id_format: '{{name}}'
+    id_format: 'projects/{{project}}/locations/{{region}}/workflows/{{name}}'
     exclude_import: true
+    schema_version: 1
```

Comment:
drive-by q: do we need a state upgrader, or can we just update the value in state on next refresh? If we can't refresh, how come?

Reply:
Name is used to call the read function, if name has been set to a long-form it won't correctly call I believe?

Reply:
Got it, thanks!